optaweb-vehicle-routing-frontend/cypress/integration/fromLocationsToRoute.js

```diff
@@ -43,6 +44,7 @@ describe('Locations can be added and route is computed', () => {
   };
 
   before(() => {
+    cy.intercept('POST', '**/api/clear').as('postClear');
```

Comment:
Nice to see this! Waiting for a route was not possible with the WebSocket in the previous implementation.